pkg/domain/infra/abi/terminal/sigproxy_linux.go

```diff
@@ -16,6 +16,48 @@ import (
 // runc is using the same value.
 const signalBufferSize = 2048
 
+func ProxySignalsRemote(ctrID string, killFunc func() error) {
+	// Stop catching the shutdown signals (SIGINT, SIGTERM) - they're going
+	// to the container now.
+	shutdown.Stop() //nolint: errcheck
```

Comment:
I don't know if you need this bit, the shutdown handler should only be invoked by non-Remote Podman

Reply:
No problem. I removed that line.